node/typings/index.d.ts

```diff
@@ -1,4 +1,3 @@
-/// <reference path="globals/jsonschema/index.d.ts" />
 /// <reference path="globals/mz/index.d.ts" />
 /// <reference path="globals/node/index.d.ts" />
 /// <reference path="globals/xml2js/index.d.ts" />
```

Comment:
Yes, along with declaring the JsonSchema type directly rather than importing it from the jsonschema package's typings.

---
In reply to: [73965181](https://github.com/openT2T/opent2t/pull/15#discussion_r73965181) [](ancestors = 73965181)